### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by nmeyerhans**
Update some build docs and provide root filesystem usage details.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,17 +96,39 @@ channel on the [Firecracker Slack](https://tinyurl.com/firecracker-microvm).
...
+
+The `image` make target can be used to invoke the root image build. As
+an optimization, you can specify a local Debian package mirror via the
+`DEBMIRROR` environment variable when generating the image, e.g.
+
+    make image DEBMIRROR=http://my-local-mirror/debian
+
+**Note** The generated filesystem image will be stored in
+`tools/image-builder/rootfs.img`. In order to use this image with
+firecracker-containerd, you must pass the following parameters to the
```
Comment (mxpv) [inline README.md]: > In order to use this image with firecracker-containerd, you must pass the following parameters to .. via the `kernel_args` setting in

would it make sense to mention that we have to copy the image to a proper location first ?

Comment (samuelkarp) [reply README.md]: Either needs to be copied or you need to specify the path in the runtime configuration file.

Comment (nmeyerhans) [reply README.md]: This text is gone now, in favor [the getting started guide](docs/getting-started.md), which already documented copying the image to the final location.

**Comment by nmeyerhans** [comment]
rebased onto master and pushed.

**Pull request merged**